.github/actions/setup-build-cuda/action.yml

```diff
@@ -23,15 +23,17 @@ runs:
         import sys
         print(sys.version)
         cushort = "${{ inputs.toolkit_short_version }}"
-        TORCH_CUDA_DEFAULT = "121"  # pytorch 2.4.1
+        TORCH_CUDA_DEFAULT = "126"  # pytorch 2.6.0
```

Comment:
Note that in any case we can't land this until PyTorch 2.6.0 is fully officially out.